From d9fb73034bd7582320521c553bfda4e59b5cd7fb Mon Sep 17 00:00:00 2001
From: AItheHuman <ariiramanesh@gmail.com>
Date: Fri, 18 Sep 2020 13:59:38 -0700
Subject: [PATCH] Patching: step 2

---
 config.def.h | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/config.def.h b/config.def.h
index 1c0b587..6ca442c 100644
--- a/config.def.h
+++ b/config.def.h
@@ -1,5 +1,9 @@
 /* See LICENSE file for copyright and license details. */
 
+#include <X11/XF86keysym.h>
+static const char *upvol[]   = { "/usr/bin/pactl", "set-sink-volume", "0", "+2%",     NULL };
+static const char *downvol[] = { "/usr/bin/pactl", "set-sink-volume", "0", "-2%",     NULL };
+static const char *mutevol[] = { "/usr/bin/pactl", "set-sink-mute",   "0", "toggle",  NULL };
 /* appearance */
 static const unsigned int borderpx  = 1;        /* border pixel of windows */
 static const unsigned int snap      = 32;       /* snap pixel */
@@ -94,6 +98,9 @@ static Key keys[] = {
 	TAGKEYS(                        XK_8,                      7)
 	TAGKEYS(                        XK_9,                      8)
 	{ MODKEY|ShiftMask,             XK_q,      quit,           {0} },
+	{ 0,                            XF86XK_AudioLowerVolume,   spawn, {.v = downvol } },
+	{ 0,                            XF86XK_AudioMute, spawn,   {.v = mutevol } },
+	{ 0,                            XF86XK_AudioRaiseVolume,   spawn, {.v = upvol   } },
 };
 
 /* button definitions */
-- 
2.17.1

